src/Microsoft.AspNetCore.SignalR.Common/Protocol/HubProtocolConstants.cs

```diff
@@ -42,5 +42,10 @@ public static class HubProtocolConstants
         /// Represents the close message type.
         /// </summary>
         public const int CloseMessageType = 7;
+
+        /// <summary>
+        /// Represents the stream complete message type.
+        /// </summary>
+        public const int StreamCompleteMessageType = 8;
```

Comment:
This new message type is another thing that you're going to have to add to the spec

Reply:
will do in a future pr

Reply:
File an issue for updating the spec